elasticsearch-client/src/main/java/fr/pilato/elasticsearch/crawler/fs/client/WorkplaceSearchClientUtil.java

```diff
@@ -67,7 +67,7 @@ public static String generateDefaultCustomSourceName(String suffix) {
         return name;
     }
 
-    public static String toRFC3339(Date d) {
+    public synchronized static String toRFC3339(Date d) {
```

Comment:
That should not be part of the same PR as you already proposed that change in another PR.